pytest.ini

```diff
@@ -21,6 +21,7 @@ markers =
     workloads: workloads related tests
     flowtests: flowbased related tests
     performance: performance related tests
+    performance_extended: performance related tests for RBD_Thick
```

Comment:
`performance related tests for RBD_Thick`
This definition is misleading. we have performance_extended for any performance tests that need not be part of regular regression, but can be run at will if required.

Reply:
@keesturam changed to : 

performance_extended: non regression performance tests